client/pom.xml

```diff
@@ -16,6 +16,9 @@
     <properties>
         <project.mainClass>org.moparforia.client.Launcher</project.mainClass>
         <project.name>Playforia Minigolf Client</project.name>
+        <project.downloadDir>${basedir}/../downloads</project.downloadDir>
+        <project.jreLinkOSX>https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u252-b09.1/OpenJDK8U-jre_x64_mac_hotspot_8u252b09.tar.gz</project.jreLinkOSX>
+        <project.jreNameOSX>jdk8u252-b09-jre</project.jreNameOSX>
```

Comment:
I would change the property name to something like `<project.jre.osx.name>`

Reply:
Much better, thanks